pkg/pod/envresolve/env.go

```diff
@@ -61,17 +63,26 @@ func getConfigMapRefValue(client kclientset.Interface, namespace string, store *
 }
 
 // getFieldRef returns the value of the supplied path in the given object
-func getFieldRef(obj runtime.Object, from *kapi.EnvVarSource) (string, error) {
+func getFieldRef(obj runtime.Object, from *corev1.EnvVarSource) (string, error) {
 	return fieldpath.ExtractFieldPathAsString(obj, from.FieldRef.FieldPath)
 }
 
 // getResourceFieldRef returns the value of a resource in the given container
-func getResourceFieldRef(from *kapi.EnvVarSource, c *kapi.Container) (string, error) {
-	return resource.ExtractContainerResourceValue(from.ResourceFieldRef, c)
+func getResourceFieldRef(from *corev1.EnvVarSource, c *corev1.Container) (string, error) {
+	// TODO: Fix this in upstream
```

Comment:
I think there is a /v1/resource package somewhere.

Reply:
good catch, moved to v1/resource